spec/lib/action_kit/helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe ActionKit::Helper do
+  before do
+    ENV['AK_PASSWORD'] = 'ak_password'
+    ENV['AK_USERNAME'] = 'ak_username'
```

Comment:
How do you feel about stubbing `ENV` as a general pattern for setting `ENV` in tests. `ENV` has a rock solid API (I don't see it changing anytime soon), so I think this is our cleanest option for unit testing.